lib/handlers/api/source-controller.ts

```diff
@@ -0,0 +1,42 @@
+import express from 'express';
```

Comment:
Missing the copyright header on this new file

Reply:
Good catch!